spring-cloud-gcp-data-spanner/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,4 @@
+<configuration>
+
+  <root level="warn"/>
```

Comment:
Does it STDOUT by default?

And since we are here already, I would like to share our experience: we add entries for root packages of the module we test against, something like:
```
<logger name="org.springframework.cloud.gcp.data.spanner" level="warn"/>
```
This way it is going to be pretty easy to change it to `debug` if something wrong happens.

Just an idea. Feel free to reject.